menas/src/main/resources/application.properties.template

```diff
@@ -20,12 +20,19 @@ multipart.maxFileSize=16Mb
 # Set the total request size for a multipart/form-data (default 10Mb)
 multipart.maxRequestSize=16Mb
 
+# How will users authenticate to menas. Available options: inmemory, kerberos
+za.co.absa.enceladus.menas.auth.mechanism = kerberos
```

Comment:
I don't like making Kerberos default. I think that we should have inmemory default as that is the "works out of the box" solution.